src/pages/variants/SearchOption.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { FunctionComponent } from 'react';
+import style from './SearchOption.module.scss';
+import { toKebabCase } from 'utils';
+
+export enum Criteria {
+  Variant = 'variant',
+  GENE = 'gene',
+}
+
+type SearchParams = {
+  symbols: string[];
+  locus: string;
+};
+
+const SQUARE_ABBREVIATION = {
+  [Criteria.Variant]: { label: 'VR' },
+  [Criteria.GENE]: { label: 'GN' },
+};
+
+type OwnProps = {
+  criteria: Criteria;
+  searchParams: SearchParams;
+};
+
+const generateSquareColorCss = (criteria: Criteria) => {
+  if (criteria === Criteria.Variant) {
+    return style.leftSquareBgColorVariant;
+  } else if (criteria === Criteria.GENE) {
+    return style.leftSquareBgColorGene;
+  }
+  return style.leftSquareBgColorVariant;
+};
+
+const SearchOption: FunctionComponent<OwnProps> = (props) => {
+  const { criteria, searchParams } = props;
+  const { symbols, locus } = searchParams;
+  return (
+    <div className={style.container}>
```

Comment:
two layer of StackLayout here, it will remove the need to add custom css for the flex stuff.
If you find issues with the stackLayout, please provide feedback or update to make them better.